php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1499,6 +1499,35 @@ abstract class NDB_BVL_Instrument extends NDB_Page
         return null;
     }
 
+    /**
+     * Determines what the data entry completion progress percentage is
```

Comment:
Can you update this comment to explain what the definition of "completion progress percentage" is in English so that it can be verified that that's what the code is doing? Also, there's a period missing at the end of the sentence.